tests/python/pants_test/backend/jvm/tasks/test_checkstyle_integration.py

```diff
@@ -61,53 +63,93 @@ def test_config_invalidates_targets(self, cache_args):
 
```

Comment:
This file should be completely restored I think? I removed the `_temporary_buildroot` method in master intentionally. If you need edits to get this passing, you'll want to make them in the base class.

Reply:
Agreed, fully-restored.